cogs/weather.py

```diff
@@ -0,0 +1,54 @@
+import os
+import discord
+from discord.ext import commands
+
+
+class Weather(commands.Cog):
+
+    WEATHER_API_KEY = os.environ["open_weather_map_key"]
+
+    def __init__(self, client):
+        self.client = client
+        print("Loading Command: Weather")
+
+    # Events
+    @commands.Cog.listener()
+    async def on_ready(self):
+        print("Bot is Ready")
```

Comment:
Sorry I didn't catch this before, but this not actually needed here in this cog (nor is it really required in the Roll command cog). It does, however, illustrate how you can make a bot do things based on a certain event. This is how you would write a custom "greeting", for example.

Let's not worry about it right now. I want this to move forward. But lets just keep in mind that we may want to remove these pointless events in the future, but keep in mind that events can be a useful tool for future functionality.